test/AspNetCoreModuleTests/AspNetCoreModuleTests.vcxproj

```diff
@@ -1,178 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why was this deleted?

Reply:
Unused project that should have been deleted a while ago. I could put it in a separate PR if you really want it to.

Reply:
Nope, if it's intentional it's fine.